docs/references/strategies/languages/scala/sbt.md

```diff
@@ -16,7 +16,8 @@ While the other analysis strategies for `gradle` and `maven` offer some scala pr
 
 ## Project Discovery
 
-Directories that contain `build.sbt` files are treated as sbt projects.
+Directories that contain `build.sbt` files are treated as sbt projects. `fossa-cli` subsequently executes, `sbt --no-colors --batch makePom` command, 
+to infer associated targets. 
```

Comment:
What does this mean to a user?

Consider something that explains this more clearly, e.g. something like the below:
> Directories that contain `build.sbt` files are treated as sbt project entry points.
>
> After finding an sbt entry point, FOSSA CLI then runs
> `sbt --no-colors --batch makePom` to infer the target to scan.
>
> Each target listed by the entrypoint is a different target scanned by FOSSA.